python/py-pyx/Portfile

```diff
@@ -21,20 +21,22 @@ long_description    PyX is a ${description}. It combines an abstraction of the P
 
 homepage            https://pyx-project.org/
 
-checksums           rmd160  61ec70ac7fb7b7b8a6fdac5cdb5c9328775c7018 \
-                    sha256  0fc3b00c5e7fb6f4aefbf63b95f624297dde47700a82b8b5ad6ebb346b5e4977 \
-                    size    2559840
+checksums           rmd160  f744331f506969deb31a529cdaccda8f2809d63d \
+                    sha256  4d8e3e471cd3e9a9bd13d5086cdf7c0af1b0c3f3e195e74f5f63318dc40a66d8 \
+                    size    626703
 
 # Note: py27-scapy depends on py27-pyx
-python.versions     27 37 38 39 310
+python.versions     27 37 38 39 310 311 312
 
 if {${name} ne ${subport}} {
     if {${python.version} == 27} {
         version     0.12.1
-        revision    1
+        revision    0
```

Comment:
the `revision` field should not be reset to `0` here

Reply:
Thanks for catching side effect of `port bump`